docs/pages/integrations.md

```diff
@@ -1,4 +1,4 @@
-# Integrations
+# Deployments
```

Comment:
links to the haystack_integration.ipynb is not working

Reply:
All the links to the notebooks are not working as the root is gh pages and not the repo aymore

Reply:
Fixed all links